### ISSUE TITLE
#-874179437------------- All page content should be contained by landmarks

### ISSUE THREAD

**Issue by khyatit13** [khyatit13/Coding-practice#1840]
<strong>PageName:</strong> FeedDetailsPage  

<strong>Source:</strong>`<header class="header"><slot name="title"></slot><slot name="subtitle"></slot></header>`

<strong>Element Location: </strong>.sc-cYYuRe > calcite-card,header

<strong>Impact: </strong>moderate

<strong>Failure Summary:</strong> Fix any of the following:
  Some page content is not contained by landmarks

<strong>Tags:</strong> cat.keyboard,best-practice

<strong>HelpUrl:<strong>https://dequeuniversity.com/rules/axe/4.6/region?application=axeAPI
<strong>Screenshot:<strong>`C:\Automation\Workspace\Velocity\Screenshot\FeedDetailsPage\-874179437_All page content should be contained by landmarks.png`
